ts/src/profiler.ts

```diff
@@ -267,6 +267,7 @@ export class Profiler extends common.ServiceObject {
       uri: '/profiles',
       body: reqBody,
       json: true,
+      timeout: 60 * 60 * 1000,
```

Comment:
It would deserve a comment about why the timeout is overridden from the default.

Reply:
Done.